module/VuFind/src/VuFind/Content/Covers/Google.php

```diff
@@ -37,9 +39,9 @@
  * @link     https://vufind.org/wiki/development Wiki
  */
 class Google extends \VuFind\Content\AbstractCover
-    implements \VuFindHttp\HttpServiceAwareInterface
+    implements \VuFind\Http\CachingDownloaderAwareInterface
```

Comment:
I don't think we set up "magic injection" for this interface, so I'm pretty sure the factories for both of these handlers will need to be updated to inject the caching downloader.

Reply:
Done!